py2pack/__init__.py

```diff
@@ -322,14 +323,23 @@ def main():
 
     # set HTTP proxy if one is provided
     if args.proxy:
+        global pypi
+
+        # add schema to url if it is not there
+        url = args.proxy
+        if "//" not in url:
+            url = "//{}".format(url)
```

Comment:
Yup, [this is valid](https://tools.ietf.org/html/rfc1738#section-3.1). We actually don't need a schema, only netloc and port.